Executable/Injector.cs

```diff
@@ -10,6 +10,7 @@ internal class Injector
     {
         private readonly string gameDirectory;
         private readonly string managedDirectory;
+        private readonly string globalgamemanagers;
```

Comment:
Should be `globalGameManagers`

Reply:
I named the variable lowercase because the actual file name is lowercase too.